index.html

```diff
@@ -2142,11 +2139,12 @@ <h3>Retrieve Verification Method</h3>
 <a href="#INVALID_VERIFICATION_METHOD">INVALID_VERIFICATION_METHOD</a>.
           </li>
           <li>
-If <var>verificationMethod</var> is not associated with the array of
-<var>vmPurposes</var> in the <var>controllerDocument</var>, either by reference
-(URL) or by value (object), an error MUST be raised and SHOULD convey an error
-type of <a href="#INVALID_PROOF_PURPOSE_FOR_VERIFICATION_METHOD">
-INVALID_PROOF_PURPOSE_FOR_VERIFICATION_METHOD</a>.
+If <var>verificationMethod</var> is not associated with the
+<var>verificationRelationship</var> array in the <var>controllerDocument</var>,
```

Comment:
Per a private discussion with @manu, there is no `verificationRelationship` data structure in the document.  Yes, there are Verification Relationships represented in the document, but the current wording will cause people to look for the data structure and fail to find it.